lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -956,7 +973,148 @@ section.instructor-dashboard-content-2 {
       }
     }
   }
+
+  /*
+   * Begin additional/override styles for cohort management.
+   * Placed for merge, but will need to be cleaned up and
+   * refactored in this stylesheet.
+   */
+
+  .has-other-input-text { // Given to groups which have an 'other' input that appears when needed
+    display: inline-block;
+
+    label {
+      display: inline-block;
+    }
+
+    .input-group-other {
+      display: inline;
+      position: relative;
+      overflow: auto;
+      width: 100%;
+      height: auto;
+      margin: 0 0 0 $baseline;
+      padding: inherit;
+      border: inherit;
+      clip: auto;
+
+      &.is-hidden {
+        display: none;
+      }
+
+      .input-cohort-group-association {
+        display: inline;
+      }
+    }
+  }
+
+  .cohort-management-nav {
+    margin-bottom: $baseline;
+  }
+
+  .cohort-management-settings {
+    @include clearfix();
+    background: $gray-l5;
+
+    .cohort-management-group-header {
+      padding: 0;
+      border-bottom: 0;
+
+      .group-header-title {
+        padding: $baseline;
+        margin-bottom: 0;
+        border-bottom: 1px solid $gray-l3;
+        text-transform: none;
+        letter-spacing: normal;
+      }
+
+      .cohort-management-group-setup {
+        padding: $baseline;
+
+        .cohort-management-group-text {
+          display: inline-block;
+          width: flex-grid(9);
+        }
+
+        .cohort-management-group-actions {
+          display: inline-block;
+          width: flex-grid(3);
+          vertical-align: top;
+          text-align: right;
+        }
+      }
+    }
+
+    .cohort-management-details,
+    .cohort-management-group-add {
+      border-top: ($baseline/5) solid $gray-l4;
+      background: $white;
+
+      .cohort-management-settings {
+        margin-bottom: 0;
+        border-top: 0;
+        background: $white;
+      }
+
+      .cohort-details-name {
+        @extend %t-action1;
+        display: block;
+        width: 100%;
+        padding: ($baseline/2);
+        margin-bottom: ($baseline*2);
+      }
+
+      .cohort-section-header {
+        margin-top: ($baseline*1.5);
+        padding: $baseline 0 ($baseline/2) 0;
+      }
+
+      .cohort-section-header > .form-field {
+        padding-bottom: $baseline;
+      }
+    }
+  }
+
+  .wrapper-tabs { // This applies to the tab-like interface that toggles between the student management and the group settings
+    @extend %ui-no-list;
+    @extend %ui-depth1;
+    position: relative;
+    top: (($baseline/5));
+    padding: 0 $baseline;
+
+    .tab {
+      position: relative;
+      display: inline-block;
+
+      a {
+        display: inline-block;
+        padding: ($baseline - 5);
+        -webkit-transition: none;
```

Comment:
Please use the [Bourbon transition mixin](http://bourbon.io/docs/#transitions) here.